python/labbox_ephys/extractors/bandpass_filter.py

```diff
@@ -1,11 +1,24 @@
+import spikeextractors as se
 from .filterrecording import FilterRecording
 import numpy as np
 from scipy import special
+import scipy.fft
+import math
 
 
 class BandpassFilterRecording(FilterRecording):
-    def __init__(self, *, recording, freq_min, freq_max, freq_wid):
-        FilterRecording.__init__(self, recording=recording, chunk_size=3000 * 10)
+    def __init__(self, *, recording: se.RecordingExtractor, freq_min, freq_max, freq_wid):
+        self._padding = 3000
+        target_ram = 100 * 1000 * 1000
```

Comment:
Target number of bytes of RAM in a chunk

Reply:
Yeah... I don't really know how to set it.

To consider: FFT scales as n*log(n), so it's not quite linear in n. Therefore there is a penalty for larger chunks. On the other hand there are other reasons why too small chunks are less efficient. So there is a sweet spot, and this was my best attempt. This can be improved, but not sure how much we will gain.